DecipherNow/gm-fabric-dashboard
# Feat: new threads toolbar

Resolves #610 

## Proposed Changes

  - Updated threads view with the new TableToolbar component based on the mock:

![image](https://user-images.githubusercontent.com/22768918/33624639-01bd99a8-d9c3-11e7-9d26-586f17bf5d7c.png)
  - Deleted old ThreadsTableToolbar
  - Updated tests
  - Completed cross browser testing
  
## How to Test

  - Navigate to JVM > Threads. Test that Group By state renders the appropriate headers. 
Note that this doesn't yet line up with the mock because I will be implementing the new column order in an upcoming PR (see issue #613).

![image](https://user-images.githubusercontent.com/22768918/33624834-96452dfc-d9c3-11e7-9da6-6f238f23c211.png)
  - Test that sorting functions correctly and toggles asc/desc
  - Test that searching functions correctly
